.changeset/weak-buttons-play.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-gocd': minor
```

Comment:
Think this is a patch change right? As it's not breaking change and the plugin is still <1.0.0?

Reply:
I got in doubt, since this does not "break API" but does change the UI behavior, by introducing that top UI Card. I will alter to point to a `patch` 👍